### PR TITLE
Fixes #2431 - Adds changelog title normalization.

### LINKED ISSUE #2431: [changelog.py][feature-request] Rewrite issue commit message separator
Our commit messages follow a convention:

`Issue #NNNN{separator}{commit_message}`

screen shot 2018-04-27 at 4 52 31 pm

We're not consistent (and I don't think it's worth caring about in a PR or worth blocking over), but it would be cool if changelog.py could normalize this so the CHANGELOG at least looked the same.

Most people seem to use `Fixes #NNNN - {commit_message}`, so that seems like the path of least resistance.

### PR DESCRIPTION
**Pull request by karlcow**
r? @miketaylr

### CODE REVIEW

**Comment by miketaylr** [comment]
a pep8 error:

```
$ pep8 --ignore=E402 webcompat/ tests/ config/secrets.py.example
tests/unit/test_tools_changelog.py:31:13: E123 closing bracket does not match indentation of opening bracket's line
tests/unit/test_tools_changelog.py:32:13: E123 closing bracket does not match indentation of opening bracket's line
```

File: tests/unit/test_tools_changelog.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
```
Comment (miketaylr) [inline tests/unit/test_tools_changelog.py]: thank you for tests here

**Comment by miketaylr** [comment]
(feel free to merge once the pep8 error is fixed)

**Comment by karlcow** [comment]
hmmm I need to fix my sublimetext linter. I'm having too many of these lately. Thanks for catching this one.

**Comment by karlcow** [comment]
ok fixed the mistake.

```
🐝 06:04:22 ~/code/webcompat.com
→ pep8 --ignore=E402 tests/
🐝 06:04:50 ~/code/webcompat.com
→ nosetests tests/unit/test_tools_changelog.py
..
----------------------------------------------------------------------
Ran 2 tests in 0.001s

OK
```

Let's run travis and I will merge. Thanks @miketaylr

**Pull request merged**